src/gen-xml.ts

```diff
@@ -622,6 +622,25 @@ function slideObjectToXml(slide: PresSlide | SlideLayout): string {
 				strSlideXml += '  <a:ext cx="' + width + '" cy="' + height + '"/>'
 				strSlideXml += ' </a:xfrm>'
 				strSlideXml += ' <a:prstGeom prst="' + (rounding ? 'ellipse' : 'rect') + '"><a:avLst/></a:prstGeom>'
+				// UGH: COPY-PASTA FROM SHAPE
+				// EFFECTS > SHADOW: REF: @see http://officeopenxml.com/drwSp-effects.php
+				if (slideItemObj.options.shadow) {
```

Comment:
Here's the `if` clause, which checks if shadow is defined.

If we leave shadow `undefined`, no shadow `xml` should be added